templates/unbound.conf.erb

```diff
@@ -11,6 +11,13 @@ server:
 <% else -%>
   use-syslog: yes
 <% end -%>
+<% if @do_not_query_address -%>
+<%- @do_not_query_address.each do |int| -%>
+  do-not-query-address: <%= int %>
+<%- end -%>
+<% elsif @do_not_query_address != '' -%>
```

Comment:
this will never be a string unless you change the type definition to `Optional[Variant[Array,String]] $do_not_query_address` 
personally i would keep things as they are so it only supports arrays and remove this clause

Reply:
Thanks for pointing that out.